content/articles/blog/drizzle.md

```diff
@@ -0,0 +1,247 @@
+
+And yet, if you’re an open source software developer, you’re probably
+using something, every single day, that came out of Drizzle. And that
+something isn’t even software.
+
+## Drizzle’s history, a very brief summary
+
+Drizzle started as an attempt to refactor MySQL, and was originally
+driven by Brian Aker, together with a small team of engineers at Sun
+(which had then-recently acquired MySQL), in the first half of 2008. A
+skunk works project that flew under the radar — to put it charitably —
+at Sun, Drizzle was publicly announced at O’Reilly OSCON of that year.
+There are a couple of videos floating around from that event ([from
+the keynotes](https://youtu.be/9DuJFUnxg7k), and [from a booth
+presentation](https://youtu.be/2tO7_Ozr-9U)) that are both… well, go
+and see for yourself. The aforementioned Stewart Smith did a very
+entertaining talk at linux.conf.au some five years later that covers
+those events, which you can watch from [the official Linux Australia
+mirror](https://mirror.linux.org.au/pub/linux.conf.au/2014/Wednesday/28-Past_Present_and_future_of_MySQL_and_variants_-_Stewart_Smith.mp4),
+or from a [YouTube
+upload](https://www.youtube.com/watch?v=6Uv9vcb4SdA).
+
+There’s also an interesting old blog post from MySQL co-founder Monty
+Widenius, [written in late July of
+2008](https://monty-says.blogspot.com/2008/07/what-if.html), which
+outlines the state of affairs at the time.
+
+Of course, in 2010 Oracle acquired Sun (and with it, the MySQL
+database) — and Oracle was presumably less than keen on having an
+*in-house* fork of the database technology it had just acquired. Thus,
+the Drizzle engineers found a new home at Rackspace, with the goal of
+getting Drizzle to a production-ready release. That *sort of*
+happened, and the Drizzle package even got into Debian, but after [the
+Drizzle 7.1 release in 2012](https://launchpad.net/drizzle/7.1),
+adoption did not exactly skyrocket. Development on Drizzle stagnated
+and eventually petered out.  The 7.2 release branch [never made it out
+of the alpha stage](https://launchpad.net/drizzle/7.2).
+
+Today, to the best of my knowledge, you can’t install a Drizzle
+package on any contemporary operating system. There is [no official
+Drizzle container image on Docker
+Hub](https://hub.docker.com/search?q=drizzle), no DBaaS offering based
+on Drizzle, nothing.
+
+But Drizzle left a very important legacy.
+
+## What did Drizzle do differently?
+
+In 2008, it was already common for open source software to live in
+public version-controlled repositories. But far from all of them used
+[Git](https://git-scm.com/), like the vast majority do today: some
+used [CVS](https://en.wikipedia.org/wiki/Concurrent_Versions_System)
+or [Subversion](https://en.wikipedia.org/wiki/Apache_Subversion), some
+used [Mercurial](https://en.wikipedia.org/wiki/Mercurial), and the
+[Launchpad](https://launchpad.net/) platform (which Drizzle lived on)
+used [Bazaar](https://en.wikipedia.org/wiki/GNU_Bazaar).
```

Comment:
Hi @fghaas - any reason why you're pointing to the git project home page, but refer to Wikipedia entries for the other DVCSes? Bazaar actually lives at https://bazaar.canonical.com/ these days, SVN is on https://subversion.apache.org/ and Mercurial has https://www.mercurial-scm.org/ - would you consider accepting a pull request to get this amended?